device/tt_silicon_driver.cpp

```diff
@@ -307,19 +281,17 @@ void detect_ffffffff_read(PCIDevice *dev, std::uint32_t data_read = 0xffffffffu)
 }
 
 inline void record_access (const char* where, uint32_t addr, uint32_t size, bool turbo, bool write, bool block, bool endline) {
-    LOG2 ("%s PCI_ACCESS %s 0x%8x  %8d bytes %s %s%s", where, write ? "WR" : "RD", addr, size, turbo ? "TU" : "  ", block ? "BLK" : "   ", endline ? "\n" : "" );
+    log_trace(LogSiliconDriver, "{} PCI_ACCESS {} 0x{:x}  {} bytes {} {}{}", where, write ? "WR" : "RD", addr, size, turbo ? "TU" : "  ", block ? "BLK" : "   ", endline ? "\n" : "" );
 }
 
 inline void print_buffer (const void* buffer_addr, uint32_t len_bytes = 16, bool endline = true) {
     // Prints each byte in a buffer
-    if (g_DEBUG_LEVEL > 1) {
```

Comment:
Was this always enabled previously? Was it always defined as larger than 1? 
Or is the same check hidden behind log_* calls?

Reply:
Ah I see now bellow it was passed through std::getenv("TT_PCI_LOG_LEVEL");
So now we'll use LOGGER_LEVEL=something instead of it, which looks fine.